api/transcript.js

```diff
@@ -20,7 +20,6 @@ const sample = (arr) => {
 
 const loadTranscript = () => {
   const moduleURL = new URL(import.meta.url);
-  
```

Comment:
@sampoder also wondering what precipitates this change in this PR?

Reply:
These used to back-to-back without a line in-between and I thought it made sense to keep it that